api/v1/installation_types.go

```diff
@@ -41,6 +41,9 @@ type InstallationSpec struct {
 	// Namespace (in Porter) where the installation is defined.
 	Namespace string `json:"namespace" yaml:"namespace"`
 
+	// Active specifies if the bundle should be installed.
+	Active bool `json:"active" yaml:"active"`
```

Comment:
After this PR, I'll see if there's a better way that doesn't involve replicating the fields from the installation document used by porter. I think what needs to happen is splitting up Porter's document like the k8s manifests are:

Installation
* InstallationSpec (embedded so it's top level)
* InstallationStatus

Then we can embed a porter.InstallationSpec type in ours here and not have to duplicate everything.